src/diff/props.js

```diff
@@ -97,6 +97,12 @@ function setProperty(dom, name, value, oldValue, isSvg) {
 		let useCapture = name !== (name = name.replace(/Capture$/, ''));
 		let nameLower = name.toLowerCase();
 		name = (nameLower in dom ? nameLower : name).slice(2);
+		
+		// Fix onInput not firing on select/radio/checkbox in Edge <= 18.
+		if (name === 'input' && (
+			type === 'select' ||
+			type === 'input' && /rad|check/.test(props.type)
```

Comment:
```suggestion
			type === 'input' && /rad|check/.test(type)
```

I guess @cristianbote is right here :)

Reply:
I was thinking this is what @developit wanted but, since the first condition checks that `type` is strictly equal to 'input' the next condition will never be true, isn't it? Cause that'll end-up with `/rad|check/.test('input')` which is not true.

Reply:
the `type` in scope right now is the vnode type (html tag / component)

we need `props.type`

Reply:
Oh, that means that `value` should be there instead.

Reply:
if you're talking about the `value` in scope, no
that's the event listener function

Reply:
You are right, I've added a solution below.